hack/godep-save.sh

```diff
@@ -31,4 +31,4 @@ REQUIRED_BINS=(
   "./..."
 )
 
-"${GODEP}" save -t "${REQUIRED_BINS[@]}"
+GOPATH=$GOPATH:$GOPATH/src/k8s.io/kubernetes/staging "${GODEP}" save -t "${REQUIRED_BINS[@]}"
```

Comment:
Only godep. We create symlinks in vendor/ to the staging dirs. So other tools like go-build have no problems.

This symlink business might go away when we extend our tooling to handle upstream commits over multiple repos. But as that is not trivial, we try to postpone that.